libsrc/target/m100/stdio/generic_console.asm

```diff
@@ -0,0 +1,62 @@
+
+
+                SECTION         code_clib
+
+                PUBLIC          generic_console_cls
+                PUBLIC          generic_console_printc
+                PUBLIC          generic_console_scrollup
+                PUBLIC          generic_console_set_ink
+                PUBLIC          generic_console_set_paper
+                PUBLIC          generic_console_set_attribute
+		PUBLIC		generic_console_xypos
+
+
+                EXTERN          CONSOLE_COLUMNS
+                EXTERN          CONSOLE_ROWS
+
+                ;EXTERN          generic_console_font32
+                ;EXTERN          generic_console_udg32
+		;EXTERN		generic_console_flags
+		;EXTERN		conio_map_colour
+
+
+generic_console_set_paper:
+generic_console_set_attribute:
+	ret
+
+
+generic_console_set_ink:
+	ret
+
+generic_console_scrollup:
+	call $45ED	; SCROLL_ALTLCD
```

Comment:
scrollup needs to preserve bc,de - does SCROLL_ALTLCD do that?

Reply:
Looks like a valid hint.  I'm still improving the whole thing,  e.g. locations vary depending on the ROM versions, etc..